web/src/main/java/org/wso2/testgrid/web/operation/TimeLimitGetter.java

```diff
@@ -0,0 +1,72 @@
+package org.wso2.testgrid.web.operation;
+
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.influxdb.dto.Query;
+import org.influxdb.dto.QueryResult;
+import org.influxdb.impl.InfluxDBResultMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.testgrid.common.TestGridConstants;
+import org.wso2.testgrid.common.config.ConfigurationContext;
+
+import java.util.List;
+
+/**
+ * This class is to get the start time and end time of the test run. These values are used to set the time period of
+ * Grafana dashboard
+ */
+public class TimeLimitGetter {
```

Comment:
Since this is Grafana specific, shall we name the class implying that as well?
Ex: `GrafanaTimeLimitGetter` 
Or else if we have multiple classes relates to grafana dashboard, we might consider move these classes into a separate directory something like,
_web/src/main/java/org/wso2/testgrid/web/operation/grafanaDashboard/_